app.js

```diff
@@ -136,17 +135,10 @@ if (Config.crashguard) {
 
 global.Sockets = require('./sockets');
 
-exports.listen = function (port, bindAddress, workerCount) {
-	Sockets.listen(port, bindAddress, workerCount);
-};
-
 if (require.main === module) {
-	// if running with node app.js, set up the server directly
-	// (otherwise, wait for app.listen())
+	// Do not set up the server during unit tests.
```

Comment:
Would something like `Set up the server directly only when running as the main module. Otherwise, in unit tests or other cases where app.js is required, wait for app.listen()` work?